src/about/donor_grants/DonorGrantsContainer.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './DonorGrantsContainer.css'
+import Contributors from './Contributors'
+
+class DonorGrantsContainer extends React.Component {
+
+  render(){
+    return (
+      <div className="container donor-grants-container">
+        <h2>Donors & Granters</h2>
+        <p>Your support means a lot to us. With your help, we're able to accelerate our 
+        work on sharding, furthering our goal to scale the Ethereum blockchain for all. We have set 
+        up a multisig smart contract to accept donations.</p>
+        <a href="https://etherscan.io/address/0x9B984D5a03980D8dc0a24506c968465424c81DbE">
+          Address: 0x9B984D5a03980D8dc0a24506c968465424c81DbE
```

Comment:
can we add ens here too? it's prysmatic.eth

Reply:
Can we punt that to another pull request? 
The objective here is to add donors and it's been outstanding for almost 3 weeks. 

@julientregoat if you would like to fix this now then go ahead, otherwise I'd like to merge these changes as soon as we can.

Reply:
Let's do that in a different PR.